src/macros.jl

```diff
@@ -766,12 +766,12 @@ that either `func` or `set` will be some custom type, rather than e.g. a
 `Symbol`, since we will likely want to dispatch on the type of the function or
 set appearing in the constraint.
 
-For extensions that need to create constraints with more information than just 
-`func` and `set`, an additional positional argument can be specified to 
-`@constraint` that will then be passed on `build_constraint`. Hence, we can 
-enable this syntax by defining extensions of 
-`build_constraint(_error, func, set, my_arg; kw_args...)`. This produces the 
-user syntax: `@constraint(model, ref[...], expr, my_arg, kw_args...)`. 
+For extensions that need to create constraints with more information than just
```

Comment:
Do this in a separate PR, this does not need to wait until v0.22

Reply:
oops. I think this just got caught up in the formatting.